terraform/shared/modules/ecr/ecr.tf

```diff
@@ -19,10 +21,10 @@ resource "aws_ecr_lifecycle_policy" "gdx_data_share_poc" {
     "rules": [
         {
             "rulePriority": 1,
-            "description": "Keep last 3 tagged images",
+            "description": "Keep last 3 tagged dev images",
             "selection": {
                 "tagStatus": "tagged",
-                "tagPrefixList": ["dev", "demo", "prod"],
+                "tagPrefixList": ["demo"],
```

Comment:
dev vs demo? I appreciate they're non-obvious, but I'm not sure they're that non-obvious?

Reply:
Typo by a slow oskar